Detectors/FIT/FDD/dcsmonitoring/workflow/fdd-dcs-sim-workflow.cxx

```diff
@@ -36,6 +37,13 @@ o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext co
   // dphints.emplace_back(o2::dcs::test::DataPointHint<uint>{"FDD/PM/SIDE_C/LAYER1/PMT_1_[0..3]/ADC[0,1]_BASELINE", 30, 150});
   // dphints.emplace_back(o2::dcs::test::DataPointHint<uint>{"FDD/PM/SIDE_A/LAYER2/PMT_2_[0..3]/ADC[0,1]_BASELINE", 30, 150});
   // dphints.emplace_back(o2::dcs::test::DataPointHint<uint>{"FDD/PM/SIDE_A/LAYER3/PMT_3_[0..3]/ADC[0,1]_BASELINE", 30, 150});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger1_Central/CNT_RATE", 0, 5000000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger2_SemiCentral/CNT_RATE", 0, 5000000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger3_Vertex/CNT_RATE", 0, 5000000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger4_OrC/CNT_RATE", 0, 5000000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger5_OrA/CNT_RATE", 0, 5000000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Background/[0..9]/CNT_RATE", 0, 50000});
+  // dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Background/[A,B,C,D,E,F,G,H]/CNT_RATE", 0, 50000});
```

Comment:
why are those commented? If not needed, can they be removed? Same comment for other FIT detectors.

Reply:
Just to make local testing a bit easier, it's less stuff to keep track of. If one works they would (probably) all work. But I uncommented all of them, one can comment out as needed instead. For sure nicer this way.